library/azure_rm_cosmosdbaccount.py

```diff
@@ -0,0 +1,479 @@
+            self.log("Database Account instance unchanged")
+            self.results['changed'] = False
+            response = old_response
+
+        if self.state == 'present':
+            self.results.update(self.format_item(response))
+        return self.results
+
+    def create_update_databaseaccount(self):
+        '''
+        Creates or updates Database Account with the specified configuration.
+
+        :return: deserialized Database Account instance state dictionary
+        '''
+        self.log("Creating / Updating the Database Account instance {0}".format(self.name))
+
+        try:
+            response = self.mgmt_client.database_accounts.create_or_update(resource_group_name=self.resource_group,
+                                                                           account_name=self.name,
+                                                                           create_update_parameters=self.parameters)
+            if isinstance(response, LROPoller) or isinstance(response, AzureOperationPoller):
+                response = self.get_poller_result(response)
+
+        except CloudError as exc:
+            self.log('Error attempting to create the Database Account instance.')
+            self.fail("Error creating the Database Account instance: {0}".format(str(exc)))
+        return response.as_dict()
+
+    def delete_databaseaccount(self):
+        '''
+        Deletes specified Database Account instance in the specified subscription and resource group.
+
+        :return: True
+        '''
+        self.log("Deleting the Database Account instance {0}".format(self.name))
+        try:
+            response = self.mgmt_client.database_accounts.delete(resource_group_name=self.resource_group,
+                                                                 account_name=self.name)
+        except CloudError as e:
+            self.log('Error attempting to delete the Database Account instance.')
+            self.fail("Error deleting the Database Account instance: {0}".format(str(e)))
+
+        return True
+
+    def get_databaseaccount(self):
+        '''
+        Gets the properties of the specified Database Account.
+
+        :return: deserialized Database Account instance state dictionary
+        '''
+        self.log("Checking if the Database Account instance {0} is present".format(self.name))
+        found = False
+        try:
+            response = self.mgmt_client.database_accounts.get(resource_group_name=self.resource_group,
+                                                              account_name=self.name)
+            found = True
+            self.log("Response : {0}".format(response))
+            self.log("Database Account instance : {0} found".format(response.name))
+        except CloudError as e:
+            self.log('Did not find the Database Account instance.')
+        if found is True:
+            return response.as_dict()
+
+        return False
+
+    def format_item(self, d):
```

Comment:
suggest better naming, it's hard to understand what's it's doing from naming. and is  this  really useful? one line code should be enough.

Reply:
i will rename... the function is for consistency. some modules have more things returned, some have less.

Reply:
done